stacks-signer/src/signer.rs

```diff
@@ -263,6 +255,28 @@ impl From<SignerConfig> for Signer {
         );
         let signer_db =
             SignerDb::new(&signer_config.db_path).expect("Failed to connect to signer Db");
+
+        let mut signing_round = WSTSSigner::new(
```

Comment:
Likewise, can you think of a better name than `signing_round` for this?  Maybe just `state_machine`?  We don't make a new state machine for each signing round.  And this name token is being reused elsewhere in the PR so it's not just local.

Reply:
Done. Also renamed the method `save_signing_round` to `save_signer_state`. :heavy_check_mark: